tests/Avalonia.Input.UnitTests/Avalonia.Input.UnitTests.csproj

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
No particular reason, I've stopped that conversion process after tests for `Avalonia.Controls` and `Avalonia.Layout` failed to run on .NET Core. We can address that later.